.env.example

```diff
@@ -1,67 +1,37 @@
-# - in development environment app will run without an ALCHEMY API KEY
-# - you can pick up Alchemy API Keys at https://auth.alchemyapi.io/
-REACT_APP_ETHEREUM_ALCHEMY_IDS=""
+###### Add an Alchemy key
+# You can sign up for an API key at https://auth.alchemyapi.io/
+# and provide that here for improved API speed and latency.
+REACT_APP_ALCHEMY_API_KEY=""
 
-REACT_APP_ARBITRUM_ALCHEMY_IDS=""
-REACT_APP_POLYGON_ALCHEMY_IDS=""
+##### Add an Infura key
+# You can sign up for an API key at https://infura.io/
+# and provide that here for improved API speed and latency.
+REACT_APP_INFURA_API_KEY=""
 
-# add your INFURA ID
-# - Infura IDs are only used as fallbacks (in conjunction with ALCHEMY_IDS)
-# - multiple IDS should be separated by whitespace
-# - you can pick up INFURA_IDS at https://infura.io/
-REACT_APP_INFURA_IDS=""
-
-# add a TESTNET ALCHEMY API KEY
-# - only 1 ID is required, multiple IDS should be separated by whitespace
-REACT_APP_ETHEREUM_TESTNET_ALCHEMY=""
-
-REACT_APP_ARBITRUM_TESTNET_ALCHEMY=""
-
-# if you run your own node you set your https address below
-# - multiple IDS should be separated by whitespace
-# - in Production node URLs must be secure `https://`
-# - in Development node URLs must be insecure `http://`
-REACT_APP_ETHEREUM_SELF_HOSTED_NODE="https://ipAddress:port"
-
-REACT_APP_ARBITRUM_SELF_HOSTED_NODE="https://ipAddress:port"
-REACT_APP_POLYGON_SELF_HOSTED_NODE="https://ipAddress:port"
-REACT_APP_AVALANCHE_SELF_HOSTED_NODE="https://api.avax.network/ext/bc/C/rpc"
-REACT_APP_FANTOM_SELF_HOSTED_NODE="https://rpc.fantom.network https://rpc.ftm.tools"
-
-# Segment Analytics
-# - not necessary in dev environment
+##### Add a Segment analytics key
 REACT_APP_SEGMENT_API_KEY=""
 
-# Google Analytics
+##### Add a Google Analytics key
 # - not necessary in dev environment
-REACT_APP_GA_API_KEY=""
+REACT_APP_GOOGLE_ANALYTICS_API_KEY=""
 
-# Covalent
-# - used to fetch token balances on wallet
-# - you can pick up a Covalent key at https://www.covalenthq.com/
-REACT_APP_COVALENT=""
+#### Add a Covalent key
+# Sign up for a Covalent key at https://www.covalenthq.com/
+REACT_APP_COVALENT_API_KEY=""
 
-# Zapper
-# Public key from Zapper, ONLY works on MAINNET
-# Zapper keys aren't needed unless you are trying to zap. 
-# Zaps aren't available except on mainnet
-REACT_APP_ZAPPER_API="96e0cc51-a62e-42ca-acee-910ea7d2a241"
-REACT_APP_ZAPPER_POOL="0x04f2694c8fcee23e8fd0dfea1d4f5bb8c352111f"
-
-# Olympus Give
+#### Olympus Give
 # These 2 settings should be toggled to "true" if you want to use Olympus Give features
-REACT_APP_GIVE_ENABLED="false"
+REACT_APP_GIVE_ENABLED="true"
 REACT_APP_MOCK_SOHM_ENABLED="false"
 
-
-########################
+##### Self hosted nodes
+# If you run your own node, you can provide a connection url to that node
+REACT_APP_FANTOM_SELF_HOSTED_NODE=""
+REACT_APP_POLYGON_SELF_HOSTED_NODE=""
+REACT_APP_ETHEREUM_SELF_HOSTED_NODE=""
+REACT_APP_ARBITRUM_SELF_HOSTED_NODE=""
+REACT_APP_AVALANCHE_SELF_HOSTED_NODE=""
```

Comment:
we could rename these to make it more clear... rather than `SELF_HOSTED_NODE` it could be `NODE_URL`